examples/language-modeling/run_clm.py

```diff
@@ -404,6 +404,10 @@ def main():
             config.update_from_string(model_args.config_overrides)
             logger.info(f"New config: {config}")
 
+    if hasattr(config, "activation_function"):
+        if config.activation_function in ["gelu_fast", "gelu_new", "gelu_python"] and training_args.use_fused_gelu:
+            config.activation_function = "gelu"
```

Comment:
Do we want to do this only in the `run_clm.py` example?
If yes, the argument `use_fused_gelu` should be defined in the example arguments [here](https://github.com/huggingface/optimum-habana/blob/ba79326a01d05b7f5b5771bc5d1823046838fae4/examples/language-modeling/run_clm.py#L67).
If no, we should rather do it in the GaudiTrainer.

Reply:
see my comments